apollo-router/src/warp_http_server_factory.rs

```diff
@@ -415,8 +413,13 @@ where
             let span = Span::current();
             // TODO headers
             tracing::debug_span!(parent: &span, "serialize_response").in_scope(|| {
-                serde_json::to_string(response.body())
-                    .expect("serde_json::Value serialization will not fail")
+                match response.into_body() {
```

Comment:
Not this PR: but this does seem like a trace_span() rather than a debug_span()... There's potentially a lot of data going to be generated here if someone enables debug tracing.

Reply:
Fixed